lib/rack/jsonp.rb

```diff
@@ -12,6 +12,7 @@ def initialize(app, options = {})
       @carriage_return = options[:carriage_return] || false
       @callback_param = options[:callback_param] || 'callback'
       @timestamp_param = options[:timestamp_param] || '_'
+      @callback_regex = options[:callback_regex] || /^[a-zA-Z0-9]{1,32}$/
```

Comment:
I think this needs to include `_`, which is used in the callback that we currently use.

Reply:
Ah, totally missed the internal PR for this. Feel free to ignore, this is a better default.